src/main/java/org/fcrepo/spec/testsuite/event/NotificationTest.java

```diff
@@ -0,0 +1,187 @@
+import static org.fcrepo.spec.testsuite.App.TOPIC_NAME_PARAM;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageConsumer;
+import javax.jms.TextMessage;
+
+import org.fcrepo.spec.testsuite.TestInfo;
+
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import com.jayway.jsonpath.JsonPath;
+
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+
+/**
+ * API Spec event testing.
+ *
+ * @author whikloj
+ * @since 2018-09-21
+ */
+public class NotificationTest extends AbstractEventTest {
+
+    /**
+     * Default constructor.
+     *
+     * @param adminUsername admin username
+     * @param adminPassword admin password
+     * @param username regular username
+     * @param password regular user password
+     * @param jmsBroker broker url
+     * @param queueName queue name
+     * @param topicName topic name
+     * @throws JMSException on error connecting to broker
+     */
+    @Parameters({ "param2", "param3", "param4", "param5", BROKER_URL_PARAM, QUEUE_NAME_PARAM, TOPIC_NAME_PARAM })
+    public NotificationTest(final String adminUsername, final String adminPassword, final String username,
+            final String password, final String jmsBroker, final String queueName, final String topicName)
+            throws JMSException {
+        super(adminUsername, adminPassword, username, password, jmsBroker, queueName, topicName);
+    }
+
+    /**
+     * 6.1 Notification Events
+     *
+     * @param uri the repository base uri
+     * @throws JMSException problems connecting to broker
+     * @throws InterruptedException interrupt the thread.sleep
+     */
+    @Test(groups = { "MUST" })
+    @Parameters({ "param1" })
+    public void testCatchMessage(final String uri) throws JMSException, InterruptedException {
+        final TestInfo info = setupTest("6.1",
+                "For every resource whose state is changed as a result of an HTTP operation, there MUST be a " +
+                        "corresponding notification made available describing that change.",
+                "https://fedora.info/2018/06/25/spec/#notification-events", ps);
+        // Start with a clean JMS connection.
+        resetConnection();
+        // Get a new consumer.
+        final MessageConsumer consumer = getConsumer();
+        // Assign a message bank to capture the messages.
+        consumer.setMessageListener(new MessageBank());
+        // Start listening to the broker.
+        connection.start();
+        // Do your actions.
+        final Response response = createBasicContainer(uri, info);
+        final String location = getLocation(response);
+        Thread.sleep(500);
```

Comment:
You'll be happier if you use [awaitility](https://github.com/awaitility/awaitility) instead of `Thread.sleep`.